fe/fe-core/src/main/java/com/starrocks/statistic/FullStatisticsCollectJob.java

```diff
@@ -66,12 +98,137 @@ public void collect(ConnectContext context, AnalyzeStatus analyzeStatus) throws
+            List<Expr> row = Lists.newArrayList();
+
+            String partitionName = table.getPartition(data.getPartitionId()).getName();
+
+            params.add(String.valueOf(table.getId()));
+            params.add(String.valueOf(data.getPartitionId()));
+            params.add("'" + data.getColumnName() + "'");
+            params.add(String.valueOf(db.getId()));
+            params.add("'" + tableName + "'");
+            params.add("'" + partitionName + "'");
+            params.add(String.valueOf(data.getRowCount()));
+            params.add(String.valueOf(data.getDataSize()));
+            params.add("hll_deserialize('mockData')");
+            params.add(String.valueOf(data.getNullCount()));
+            params.add("'" + data.getMax() + "'");
+            params.add("'" + data.getMin() + "'");
+            params.add("now()");
+            // int
+            row.add(new IntLiteral(table.getId(), Type.BIGINT)); // table id, 8 byte
+            row.add(new IntLiteral(data.getPartitionId(), Type.BIGINT)); // partition id, 8 byte
+            row.add(new StringLiteral(data.getColumnName())); // column name, 20 byte
+            row.add(new IntLiteral(db.getId(), Type.BIGINT)); // db id, 8 byte
+            row.add(new StringLiteral(tableName)); // table name, 50 byte
+            row.add(new StringLiteral(partitionName)); // partition name, 10 byte
+            row.add(new IntLiteral(data.getRowCount(), Type.BIGINT)); // row count, 8 byte
+            row.add(new IntLiteral((long) data.getDataSize(), Type.BIGINT)); // data size, 8 byte
+            row.add(hllDeserialize(data.getHll())); // hll, 16 kB
+            row.add(new IntLiteral(data.getNullCount(), Type.BIGINT)); // null count, 8 byte
+            row.add(new StringLiteral(data.getMax())); // max, 200 byte
+            row.add(new StringLiteral(data.getMin())); // min, 200 byte
+            row.add(nowFn()); // update time, 8 byte
+
+            rowsBuffer.add(row);
+            sqlBuffer.add("(" + String.join(", ", params) + ")");
+        }
+        flushInsertStatisticsData(context, false);
+    }
+
+    private void flushInsertStatisticsData(ConnectContext context, boolean force) throws Exception {
+        // about 17kb
+        long bufferSize = 17L * 1024 * rowsBuffer.size();
+        if (bufferSize < Config.statistics_full_collect_buffer && !force) {
+            return;
+        }
+        if (rowsBuffer.isEmpty()) {
+            return;
+        }
+
+        int count = 0;
+        int maxRetryTimes = 5;
+        StatementBase insertStmt = createInsertStmt();
+        do {
+            // don't work if not leader, because origin statement is a mock SQL and can't forward to leader
+            if (!(GlobalStateMgr.getCurrentState().isLeader())) {
+                LOG.info("Statistics collect fail because this isn't leader");
+                return;
+            }
+
+            LOG.debug("statistics insert sql : " + insertStmt.getOrigStmt().originStmt);
```

Comment:
Is necessary to print the mocked sql? It's too long. We'd better print some necessary info?

Reply:
To be honest, I didn't think too clearly, it's just used for my verification work